packages/ember-template-compiler/lib/system/compile-options.ts

```diff
@@ -80,110 +79,3 @@ interface LegacyPlugin {
 }
 
 export type LegacyPluginClass = new (env: ASTPluginEnvironment) => LegacyPlugin;
```

Comment:
I think this can be removed now. There is a referencing in a typing file I'm not sure exactly how to unravel (seems to be entangled with a `__raw` property?) but do you think it can also be removed?

Reply:
I think it can be, but because I’m a TS noob I wasn’t sure how 😓